oak_containers/proto/interfaces.proto

```diff
@@ -79,6 +97,15 @@ service Launcher {
   rpc NotifyAppReady(google.protobuf.Empty) returns (google.protobuf.Empty) {}
 }
 
+// Defines the service exposed by the orchestrator, that can be invoked by the application.
+service ControlPlane {
```

Comment:
I'm personally not convinced that control plane is the right metaphor here. I don't think it accurately describes our current system and could be misleading. 

Could we rename this service to something different, eg `OrchestratorExternal`?

Reply:
I agree, or even be more concrete and name the service something like `KeyProvisioning`.

Reply:
Incidentally, these should really be two distinct services already. Eventually, the KPS will implement one, and the Orchestrator will implement the other.

Reply:
Renamed to `KeyProvisioning`

Reply:
Can you comment on whether they should be one service or two? Eventually they will be implemented by separate components, right? So why not split them already?

Reply:
I'm not sure whether we should split different services that listen to the same entity, i.e. have multiple Orchestrator services that listen for messages from the Hostlib. We could just have one service that has all of the corresponding methods.

Reply:
That's not a very strong argument, since gRPC allows multiplexing multiple services on the same port. My point is: do we agree that once we have KPS, then we would remove `GetProvisionSecrets` from here and add it to a new service definition for the KPS? If so, we may as well do that now (and also align with Johnnie et al. about that interface). It does not have to be in this literal PR, but we need to have a single plan for how to get there.

Reply:
Yes, `GetProvisionSecrets` will be removed, but the new KPS service definition will be the same as this one, because the instances of the KPS will need to share the keys between them as well.

So in the future we'll have 2 services.
Orchestrator:
```protobuf
service KeyProvisioning {
  rpc SendProvisionSecrets(SendProvisionSecretsRequest) returns (google.protobuf.Empty) {}
}
```
KPS:
```protobuf
service KeyProvisioning {
  rpc GetProvisionSecrets(GetProvisionSecretsRequest) returns (GetProvisionSecretsResponse) {}
  rpc SendProvisionSecrets(SendProvisionSecretsRequest) returns (google.protobuf.Empty) {}
}
```